src/views/Checkout.vue

```diff
@@ -103,6 +104,8 @@ class Checkout extends Vue {
     private initialCurrencies: PublicCurrency[] = [];
     private availableCurrencies: PublicCurrency[] = [];
     private readonly isIOS: boolean = BrowserDetection.isIOS();
+    private readonly disclaimerDisabled: boolean = // disable disclaimer for voting app
+        /^https?:\/\/(www\.)?nimiq(-testnet)?\.com\/vote\b/.test(document.referrer);
```

Comment:
Seems like this would be a good case for a config setting again, like with the NIM-less checkout origins.

A referrer might not be available in all browsers though, as some browsers or browser extensions might block all referrers, or block the `path` part of a referrer.

Reply:
In the config we only have options that depend on the origin so far, therefore I was hesitant to add this option (which depends on a path) to the config to avoid having to come up with a complicated config name.
I think having a regex check here is good enough.

Do you have more insights about blocked referrers / paths?
I tested in Chrome and Firefox and I was able to read the referrer including the path.
An alternative would be to add a `disableDisclaimer` option for privileged origins to the Checkout request, further complicating that request.

Reply:
https://chrome.google.com/webstore/detail/referer-control/hnkcfpcejkafcihlgbojoidoihckciin for example is an extension for Chrome that lets people disable the referrer globally.

Reply:
Well, such extensions do not seem to be wide spread and extensions can always break things.
But I changed it nonetheless to a `disableDisclaimer` flag. In the end, I think its a little bit nicer.